src/EPL-2.0.xml

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+	<license isOsiApproved="true" licenseId="EPL-2.0"
+		name="Eclipse Public License 2.0">
+		<crossRefs>
+			<crossRef>http://www.eclipse.org/legal/epl-2.0</crossRef>
+			<crossRef>http://www.opensource.org/licenses/EPL-2.0</crossRef>
+		</crossRefs>
+		<notes></notes>
+		<titleText>
+			<p>Eclipse Public License - v 2.0</p>
+		</titleText>
+		<p>THE
+			ACCOMPANYING PROGRAM IS PROVIDED UNDER THE TERMS OF THIS ECLIPSE
+			PUBLIC LICENSE ("AGREEMENT).
+			ANY USE, REPRODUCTION OR DISTRIBUTION
+			OF
+			THE PROGRAM CONSTITUTES
+			RECIPIENT&#039;S ACCEPTANCE OF THIS
+			AGREEMENT.
+		</p>
+		<list>
+			<item>
+				<bullet>1.</bullet>
+				DEFINITIONS
+				<p>"Contribution"
+					means:
+				</p>
+				<list>
+					<item>
+						<bullet>a)</bullet>
+						in the case of the initial Contributor, the initial content
+						Distributed under this Agreement, and
+					</item>
+					<item>
+						<bullet>b)</bullet>
+						in the case of each subsequent Contributor:
+						<list>
+							<item>
+								<bullet>i)</bullet>
+								changes to the Program, and
+							</item>
+							<item>
+								<bullet>ii)</bullet>
+								additions to the Program;
+							</item>
+						</list>
+						<p>where
+							such changes and/or additions to the Program originate
+							from
+							and
+							are Distributed by that particular Contributor. A
+							Contribution
+							"originates"
+							from a Contributor if it was added to the
+							Program by such
+							Contributor itself or anyone acting on such
+							Contributor&#039;s
```

Comment:
We would prefer UTF-8 for the apostrophe (`'`).  See #314.

Reply:
I'll remove the empty note and replace the apostrophes.

In the process of converting the HTML document into the license schema, I ended up converting a blockquote into a paragraph (p). Does it instead make more sense to using a list/item combination?

Out of curiosity, why not just wrap the existing HTML in a CDATA block? Converting to the license schema seems unnecessary at best and potentially problematic (in, e.g., cases like mine when the original license text uses a blockquote).

Reply:
When we convert the XML to the HTML for the spdx.org/licenses site, we add styling for the `<alt>` and `<optional>`.  This would be challenging to process if we were working from the original HTML.

The tool that generates the website does have hooks for passing in original HTML if we wanted to go that route - we would just lose the optional and alt highlighting.

Reply:
> Out of curiosity, why not just wrap the existing HTML in a CDATA block?

Besides the issues mentioned by @goneall, I also spent a bit of time working on XHTML embedding and couldn't get validation working.  See [here][1] and later comments in that issue mentioning “XHTML”.

> Converting to the license schema seems unnecessary at best and potentially problematic (in, e.g., cases like mine when the original license text uses a blockquote).

EPL-2.0 seems to use `<blockquote>` for [Exhibit A][2], but I'm a bit confused by its placement:

```console
$ curl -s 'https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.html' | grep -B6 -A2 '<blockquote>'
    <h2 id="exhibit-a">Exhibit A &ndash; Form of Secondary Licenses Notice</h2>
    <p>&ldquo;This Source Code may also be made available under the following 
        Secondary Licenses when the conditions for such availability set forth 
        in the Eclipse Public License, v. 2.0 are satisfied: {name license(s),
        version(s), and exceptions or additional permissions here}.&rdquo;
    </p>
    <blockquote>
      <p>Simply including a copy of this Agreement, including this Exhibit A
        is not sufficient to license the Source Code under Secondary Licenses.
```

I'd have expected:

```xml
<blockquote>
  <p>This Source Code may also be made available under the following 
    Secondary Licenses when the conditions for such availability set forth 
    in the Eclipse Public License, v. 2.0 are satisfied: {name license(s),
    version(s), and exceptions or additional permissions here}.
  </p>
</blockquote>
<p>Simply including a copy of this Agreement, including this Exhibit A…
```

Are you sure the upstream blockquote is in the right place?

[1]: https://github.com/spdx/license-list-XML/issues/391#issuecomment-282795938
[2]: https://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.html#exhibit-a